src/frontend/types.ts

```diff
@@ -241,6 +241,11 @@ export interface LibraryContextType {
   setShowUpdatesOnly: (value: boolean) => void
   handleAddGameButtonClick: () => void
   setShowCategories: (value: boolean) => void
+  showAlphabetFilter?: boolean
+  onToggleAlphabetFilter?: () => void
```

Comment:
can these 2 properties ever be undefined? I think we are always passing a value for both